index.html

```diff
@@ -88,7 +89,7 @@ <h1>I'm Jonathan Glad to see you</h1>
 
   <section class="project-container">
     <div class="project-card">
-      <img src="./assets/Snapshoot Portfolio-2.png" alt="project card image" />
+      <img src="./assets/Snapshoot Portfolio.png" alt="project card image" />
```

Comment:
- [ ] For all your project cards, they are not fully matching the figma design. Please take a look at the images below to compare and contrast and the fix the issues by tweaking your margin or padding to match the design.

**figma**
![des1](https://user-images.githubusercontent.com/99656687/220836572-4a9d0c02-91cb-4e3a-b95e-8da04be1c73d.png)
**Your implementation**
![des2](https://user-images.githubusercontent.com/99656687/220836603-ff644e34-e8fa-4d69-b4a6-f750ac2aeb93.png)